capstone/src/main/java/com/google/sps/servlets/PollServlet.java

```diff
@@ -78,4 +78,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(optionEntity);
   }
+
+  /**
+   * Handles error for Java Servlet and displays that something went wrong.
+   *
+   * @param response    HttpServletResponse
+   * @param errorString error message
+   * @throws IOException exception thrown when cannot write to file
+   */
+  private void sendError(HttpServletResponse response, String errorString) throws IOException {
+    response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Cannot parse to long.");
+    response.getWriter().print("<html><head><title>Oops an error happened!</title></head>");
+    response.getWriter().print("<body>Something bad happened uh-oh!</body>");
+    response.getWriter().println("</html>");
+  }
```

Comment:
https://github.com/googleinterns/step100-2020/blob/master/capstone/src/main/java/error/ErrorHandler.java

Can you just use the `ErrorHandler` class you created instead of reimplementing this logic?

Reply:
On a related note, can we make the sendError() method static? So we don't have to instantiate  an ErrorHandler every time we want to send an error message

Reply:
Actually I think it would make sense to make it static